src/module.js

```diff
@@ -218,10 +218,8 @@ export default class DailyIframe extends EventEmitter {
               edge: "<0",
             }
           }),
-          isScreenShareSupported = isValidBrowser && 
-            // The following is the exact screen share compatibility check the site uses
-            navigator.userAgent.match(/Chrome\//) && 
-            !navigator.userAgent.match(/Edge\//);
+          isScreenShareSupported = !!(isValidBrowser && 
+            navigator && navigator.mediaDevices && navigator.mediaDevices.getDisplayMedia);
```

Comment:
does electron use daily-js at all? should there be an electron check here?

Reply:
nm. you're not calling getDisplayMedia. got my wires crossed.

Reply:
it is admittedly a bit weird. this check will return true for electron (as it should) because `getDisplayMedia` is defined in electron (even though it doesn't work when invoked). relying on that fact is admittedly odd.